src/test/test_dir.c

```diff
@@ -6654,6 +6654,54 @@ test_dir_dirserv_router_get_status(void *arg)
   routerinfo_free(ri);
 }
 
+static void
+test_dir_dirserv_would_reject_router(void *arg)
```

Comment:
What happens when dirserv_would_reject_router() is called on an empty fingerprint list?
What should happen?

Reply:
I fixed it. Both the values return 0 for RSA and ed25519.

Reply:
e312885f7079ad681bbfafe5d220aeade597c8a8